src/irm.cpp

```diff
@@ -122,6 +122,8 @@ usage() {
         " ",
         "Options are:",
         " -f  force - Immediate removal of data objects without putting them in trash.",
+        "             The command does not show an error message or return a non-zero ",
+        "             status code for non-existent data objects.",
```

Comment:
This is slightly incorrect (based on the implementation).

How about ...
```
Ignores non-existent data objects and collections.
```

Reply:
Thank you, that is indeed better. I've changed the text.